tracer/test/snapshots/Security.AspNetCore5AsmToggle._.verified.txt

```diff
@@ -0,0 +1,49 @@
+﻿[
```

Comment:
Since you didn't touch this file, I think this snapshot may also be unnecessary and deleted

Reply:
removed in b9a46c5f6